### DISCUSSION TITLE
Error Messages with React Calendar

### DISCUSSION THREAD

**Discussion by therealchrismcfly** [neuefische/questions#109] [Web] [answered]
Hallo hallo,

ich habe diesen Kalender (React Calendar) eingebaut: 

<img width="535" alt="Bildschirmfoto 2022-08-25 um 11 54 22" src="https://user-images.githubusercontent.com/107792731/186635271-0d90a230-822c-4056-8a1f-b12fcc33d456.png">

Bekomme jedoch die drei folgenden Fehlermeldungen:

<img width="958" alt="Bildschirmfoto 2022-08-25 um 11 54 35" src="https://user-images.githubusercontent.com/107792731/186635461-f397c6bf-ff50-42fc-a51b-7ab283ac4402.png">
<img width="942" alt="Bildschirmfoto 2022-08-25 um 11 54 44" src="https://user-images.githubusercontent.com/107792731/186635468-c3071cc0-5b40-4a70-8e4b-698209302c77.png">
<img width="949" alt="Bildschirmfoto 2022-08-25 um 11 54 52" src="https://user-images.githubusercontent.com/107792731/186635471-74ab562b-714a-4b63-9fbe-f77bd034cfef.png">

Meine Vermutung ist: ich muss useEffect in die Funktion The Calendar einbauen? Die große Frage ist, was genau ich dann vor das return schreibe. Hier der Link zu meinem GitHub: 

https://github.com/therealchrismcfly/capstone-project/tree/calendar

Danke schonmal für eure Hilfe :-)

**Comment by tsimvris** [comment, marked as answer]
Hi Chris,

check diese [discussion](https://github.com/orgs/neuefische/discussions/68)

**Reply by mpagels** [reply]
I checked it with your project. The solution, in the linked discussion, helps you solving the problem.

**Reply by therealchrismcfly** [reply]
it worked :-) Thanks a lot!
